pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -51,6 +55,20 @@ type HyperConvergedConfig struct {
 	NodePlacement *sdkapi.NodePlacement `json:"nodePlacement,omitempty"`
 }
 
+type HyperConvergedCertConfig struct {
+	// CARotateInterval configurated duration for CA and certificate
```

Comment:
Is there a syntax or a measure unit for this?

Reply:
string representation of time.Duration - we're just copying these to CNAO and they're doing the validation https://github.com/kubevirt/cluster-network-addons-operator#self-signed-certificates-configuration